Status: Checked in to HEAD after 6.3.

2004-11-09  Daniel Jacobowitz  <dan@debian.org>

	* dwarf2-frame.c (dwarf2_frame_state_alloc_regs): Correct allocated
	size.

Index: src/gdb/dwarf2-frame.c
===================================================================
RCS file: /cvs/src/src/gdb/dwarf2-frame.c,v
retrieving revision 1.45
diff -u -p -r1.45 dwarf2-frame.c
--- src/gdb/dwarf2-frame.c	7 Nov 2004 21:16:11 -0000	1.45
+++ src/gdb/dwarf2-frame.c	9 Nov 2004 14:42:52 -0000
@@ -162,7 +162,7 @@ dwarf2_frame_state_alloc_regs (struct dw
 static struct dwarf2_frame_state_reg *
 dwarf2_frame_state_copy_regs (struct dwarf2_frame_state_reg_info *rs)
 {
-  size_t size = rs->num_regs * sizeof (struct dwarf2_frame_state_reg_info);
+  size_t size = rs->num_regs * sizeof (struct dwarf2_frame_state_reg);
   struct dwarf2_frame_state_reg *reg;
 
   reg = (struct dwarf2_frame_state_reg *) xmalloc (size);
